mkt/latecustomization/serializers.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework import relations, serializers
+
+import mkt.carriers
+import mkt.regions
+from mkt.api.fields import SlugChoiceField
+from mkt.extensions.serializers import ExtensionSerializer
+from mkt.fireplace.serializers import FireplaceAppSerializer
```

Comment:
I thought this serializer would work but it doesn't include `regions` which is required by the transonic app widget. Can you switch this to use `mkt.webapps.serializers.SimpleAppSerializer` please?